doc/protocol.md

```diff
@@ -1071,6 +1071,7 @@ The following options can be specified in the "open" control message:
     If set `group` also has to be set.
   - `group`: a string, or an integer, the gid of the file group (`st_gid`)
     If set `user` also has to be set.
+  - `mode`:  an integer the octal representation of permission bits
```

Comment:
this text is confusing.  probably you need to put a comma in there at least, but the mention of "octal" is very weird without saying "this is usually expressed in octal, but in json it is the equivalent value in decimal" or whatever....